sklearn/preprocessing/data.py

```diff
@@ -285,8 +316,16 @@ def inverse_transform(self, X):
         check_is_fitted(self, 'scale_')
 
         X = check_array(X, copy=self.copy, ensure_2d=False)
-        X -= self.min_
-        X /= self.scale_
+
+        if len(X.shape) == 2:
```

Comment:
It's not met when the passed in array is 1-d. If it's 1-d, my code there won't work.